app/views/people/current_constituency.html.haml

```diff
@@ -0,0 +1,6 @@
+%h1= @person.full_name
+
+%h3= t('.title')
+%p= link_to(@constituency.name, party_path(@constituency.graph_id))
+%p
+  = "#{l(@constituency.seat_incumbencies.first.start_date, format: :default)} - " unless @constituency.seat_incumbencies.first.start_date.nil?
```

Comment:
You use l for dates and times rather than t (which you do for text).
See: [i18n](http://guides.rubyonrails.org/i18n.html#adding-date-time-formats))